pkgs/development/python-modules/django-bootstrap5/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+# build-system
+, hatchling
+
+# dependencies
+, beautifulsoup4
+, pillow
+, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-bootstrap5";
+  version = "23.3";
+  format = "pyproject";
+
```

```suggestion

  disabled = pythonOlder "3.X";
  
```
you should disable python3 versions older then 3.X what ever the package requires.

---

Fixed